symfit/core/minimizers.py

```diff
@@ -102,7 +102,7 @@ def execute(self, bounds=None, jacobian=None, **minimize_options):
             bounds=bounds,
             constraints=self.constraints,
             jac=self.wrapped_jacobian,
-            **minimize_options
+            options=minimize_options
```

Comment:
I disagree with this line, because there might be more kwargs then just options.

Reply:
The best alternative is the way it was :). If they ever add a kwarg to minimize, we will have to have this discussion again. I agree that having to explicitly say `options={maxiter: 1000}` in your calls sucks, but that is what #106 should solve someday.